### DISCUSSION TITLE
ValueError exception triggered in pwmerge

### DISCUSSION THREAD

**Discussion by bbinet** [ltsp/ltsp#489] [Q&A] [answered]
See screenshot below:

![shrink_IMG_20210630_182348](https://user-images.githubusercontent.com/76281/123998913-3c1fab00-d9d2-11eb-9fe1-907ebb727e17.jpg)

Do you know what's wrong with the line `bruno:x:1000` ?

**Comment by alkisg** [comment, marked as answer]
It's missing a colon:

`bruno:x:1000:`

**Reply by bbinet** [reply]
Yes, thank you!
